app/controllers/api/restful_controller.rb

```diff
@@ -92,7 +92,7 @@ def parse_date_parameters
   end
 
   def page_collection(collection)
-    collection.offset(params[:from]).limit(params[:per] || default_page_size)
+    collection.offset(params[:from]).limit(params[:per].try(:to_i) || default_page_size)
```

Comment:
I think this should be `params.fetch(:per, default_page_size)` Fetch is a great way to provide a default when the key does not exist. Also do we want to apply a fetch on the params[:from] so it's params.fetch(:from, 1) ?